<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TechNotes</class>
 <widget class="QMainWindow" name="TechNotes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TechNotes</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="servicesComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Open Services</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Closed Services</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="servicesListWidget"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QPushButton" name="exportServiceButton">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Export selected service to a plain .txt file&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Ex&amp;port Service</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="printServiceButton">
          <property name="text">
           <string>&amp;Print Service</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="deleteServiceButton">
          <property name="text">
           <string>&amp;Delete Service</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Service Information:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="firstNameLabel">
          <property name="text">
           <string>First Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="firstNameEdit">
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="placeholderText">
           <string>Enter First Name (mandatory)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Last Name: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lastNameEdit">
          <property name="placeholderText">
           <string>Enter Last Name (mandatory)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Phone Number:    </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="phoneNumberEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Enter Phone # (mandatory)</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="serviceLabel">
          <property name="text">
           <string>Services:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="servicesEdit">
          <property name="placeholderText">
           <string>List services to be performed (mandatory)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Started: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="startedDateTimeEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Due: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="dueDateTimeEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Computer Type: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="computerTypeComboBox">
          <item>
           <property name="text">
            <string>Desktop</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Laptop</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Other</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="computerName">
          <property name="text">
           <string>Computer Model: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="computerModelEdit">
          <property name="placeholderText">
           <string>Manufacturer, model, color, etc. (optional)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Location:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="locationComboBox">
            <item>
             <property name="text">
              <string>Lockup</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Tech Deck 1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Tech Deck 2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Other</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Additional Items: &lt;span style=&quot; font-style:italic;&quot;&gt;(cables, power supply, external storage, software, etc.)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="additionalItemsEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>42</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Notes:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="popOutButton">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Shows the notes for the current service in a new window.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>&amp;Pop Out</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="notesEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Current Technician:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="currentTechnicianComboBox">
          <item>
           <property name="text">
            <string>Nicholas P.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Rupinder</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nick F.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Zach</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Jason</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Jara</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Alex</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Jarrod</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Service Status: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="serviceStatusComboBox">
          <item>
           <property name="text">
            <string>Not Started</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>In Progress</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Customer Called</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Closed</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QPushButton" name="updateServiceButton">
          <property name="text">
           <string>&amp;Update Service</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="editServiceButton">
          <property name="text">
           <string>&amp;Edit Service</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="newServiceButton">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Create a new service.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>&amp;New Service</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="cancelButton">
          <property name="text">
           <string>&amp;Cancel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="applyButton">
          <property name="toolTip">
           <string>Apply changes to service</string>
          </property>
          <property name="text">
           <string>&amp;Apply</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>servicesComboBox</tabstop>
  <tabstop>servicesListWidget</tabstop>
  <tabstop>firstNameEdit</tabstop>
  <tabstop>lastNameEdit</tabstop>
  <tabstop>phoneNumberEdit</tabstop>
  <tabstop>servicesEdit</tabstop>
  <tabstop>startedDateTimeEdit</tabstop>
  <tabstop>dueDateTimeEdit</tabstop>
  <tabstop>computerTypeComboBox</tabstop>
  <tabstop>computerModelEdit</tabstop>
  <tabstop>locationComboBox</tabstop>
  <tabstop>additionalItemsEdit</tabstop>
  <tabstop>popOutButton</tabstop>
  <tabstop>notesEdit</tabstop>
  <tabstop>currentTechnicianComboBox</tabstop>
  <tabstop>serviceStatusComboBox</tabstop>
  <tabstop>exportServiceButton</tabstop>
  <tabstop>deleteServiceButton</tabstop>
  <tabstop>updateServiceButton</tabstop>
  <tabstop>newServiceButton</tabstop>
  <tabstop>applyButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
